### PR TITLE
Private events pull request

### PR DESCRIPTION
**Pull request by vichuge**
In this project, we build a site similar to a private Eventbrite which allows users to create events and then manage user signups.

### CODE REVIEW

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

### Hello @vichuge and @icebox827, you've got a great project here, well done 👍 
I like the UI, it's simple and goal-oriented. 👏 

However, there are still some changes that are necessary for this project to be approved.

**Setup:**

- [x] In your readme file intro, there are some typos, kindly correct them. "This project is ~a Evenbrite~ Lite ~wep~ app ~wich~ allow ~user~ to sign ~uu~, sign in and create events." 
#### Always use an app like Grammarly to double-check for good grammatical tenses and typos before doing that final push.

- [x] In the Usage section of your readme file, be a bit more detailed about providing users with detailed setup instructions. Use three opening and closing back-ticks (```) for code blocks... From cloning the project to setting up the database.

**Behaviour**:

- [x] There is no link for me to visit the all events page
- [x] There is no link for me to visit a single event page
- [x] I should be able to click on the users' name on the navigation bar to visit the user's profile page. But currently, I am unable to do that.
- [x] User should be able to attend a particular event and on the single event page, we should see the list of attendees

**Models:**

- [x] Upcoming events should include the current date (today). So instead d of `where(['events.date > ?', Date.today]`, it should be 
`where('date >= ?', Time.zone.now)`

- [x] The past events should not include the current date(today) since, its yet to return. So instead of `where(['events.date <= ?', Date.today])` you should have 
`where('date < ?', Time.zone.now)`

**UsersController:**

- [x] There would absolutely be no need to perform logic like this (`if event.date >= DateTime.now && @future == 0`) in your view if you are calling the scopes you created correctly for both past and previous events in the event model.
 * `@created_events = current_user.events` and for past events created by that user will be 
`@past_events = current_user.attended_events.previous`. Repeat some concept for upcoming events.

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla) or drop comments on this thread, if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

File: app/views/events/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    </div>
+    <br/>
+    <% end %>
+  </div>
+  <div class="col-md-6">
+  <h2>Future</h2>
+  <% @events_next.each do |event| %>
+    <div class="card">
+      <div class="card-body">
+        <h5 class="card-title border-bottom pb-2"><%= event.name %> </h5>
```
Comment (acushlakoncept) [inline app/views/events/index.html.erb]: I see you've created the single event page, but there is no link to it. Make `<%= event.name %>` linkable: `/events/:id`

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

### Hello @vichuge and @icebox827, Great job! 👏  your progress is commendable
Let's make a few changes, and we will be done.

- [ ]  You've created the invitations table, containing an event ID and attendee ID as well as the single event page which you are yet to provide a link for so that on all events page, I can click each and see the list of attendees. Please kindly create an `Attend Event` button for people to start attending created events 😄 

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Review by ghost — APPROVED** [review]
# Status: Approved 🎉

**Congratulations team 👏. Good luck with your next project 😃.**

**Pull request merged**